python/katana/local/_graph.pyx

```diff
@@ -367,8 +456,8 @@ cdef class Graph(GraphBase):
         return <uint64_t>self.underlying_property_graph()
 
     def global_out_degree(self, uint64_t node):
-        return len(self.edges(node))
+        return len(self.edge_ids(node))
 
-    def global_in_degree(self, uint64_t node):
-        # TODO(loc) needs shared-memory bi-directional view
-        raise NotImplementedError()
+    # def global_in_degree(self, uint64_t node):
```

Comment:
why is this now under comments?

Reply:
Ooops. I hate "this isn't implemented implementations", but that's not the point of this PR. I'll revert this.